docs/application/tizen-studio/release-notes/3-0-release-notes.md

```diff
@@ -0,0 +1,81 @@
+	-   IoT applications can now be installed without the Cloud service cetificate and key.
+-   Package Manager
+	-   Added a label warning user to terminate all Tizen Studio tasks before installing or uninstalling any package.
+
+### Deprecated
+-   NativeIDE
+	-   Deprecated Native UI Builder, Component Designer and Enventor for Tizen 5.0.
+
+### Known Issues
+-	Tizen Studio
+	-	In case of Mac, if the UI perspectives are not displayed properly after updating the Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from previous installation.
+	-	In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+-	Common
+	-   If you install the Tizen Studio in a directory that requires administrator privileges for access. For example, for C:\ProgramFiles, administrator privileges are required to run the Tizen SDK tools. The Tizen Installer and the baseline SDK Installer alerts you, if you try to install into such a directory.
+-	Web and Native IDE
+	-   From Tizen Studio 2.0 onwards, the Connection Explorer is replaced with the Device Manager, this can cause errors in the Connection Explorer view. You can fix this in two ways:
+	    -   Reset the perspective.
+            In the Tizen Studio menu, select Window > Perspective > Reset Perspective
+	    -   After updating to the Tizen Studio 2.0, run the *eclipse.exe -clean -clearPersistedState* command. Then launch the Tizen Studio normally.
+	-   You can create unit tests for Tizen 2.3.2 and higher version projects only. Now the Tizen Studio does not support unit testing for older versions.
+-	Web IDE
+	-   The preview tab in the Web Page Editor sometimes does not appear properly. Use an alternative feature, named Web SDK HTML Editor, which has enhanced features compared to the Web Page Editor. Instead of the preview tab in the Web Page Editor, use the preview feature **Ctrl + 4** of the Web SDK HTML Editor.
+	-   In Rapid Development Support(RDS) mode, the web unit test result is not updated.
+-	Certificate Manager
+	-   Overwriting a duplicate certificate profile in the migration wizard works incorrectly on macOS.
+-	Native UI Builder
+	-   If the expanded attribute in a multibutton entry component is set to false, + is displayed.
+-	Native Component Designer
+	-   The vector-type part is not supported. You cannot see the vector image and change the SVG file.
+	-   Playing sound is not supported on Windows&reg; or macOS.
+	-   The Component Designer crashes if an alias is selected as the source group of an added item.
+-	Emulator
+	-   To use the Tizen emulator, install an Intel VTx supported by the CPU, and the latest version of the graphic card driver provided by the vendor. To verify the prerequisites for the Tizen emulator, see [Prerequisites for the Tizen Studio](https://developer.tizen.org/development/tizen-studio/download/installing-tizen-studio/prerequisites).
```

Comment:
Hello, @Reni-90.
If possible please guide to use the link by md files in tizen-docs/ repository. 
As `../setup/prerequisites.md`.